src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/MinimalPullRequest.java

```diff
@@ -0,0 +1,42 @@
+package com.atlassian.bitbucket.jenkins.internal.scm;
+
+import com.atlassian.bitbucket.jenkins.internal.model.BitbucketPullRequest;
+
+import java.io.Serializable;
+
+public class MinimalPullRequest implements Serializable {
+
+    private final String fromRefDisplayId;
+    private final String fromRefId;
+    private final long pullRequestId;
+    private final String toRefDisplayId;
+    private final String toRefId;
+
```

Comment:
```suggestion
private static final long serialVersionUID = 1L;
```
So we can control the versioning of the object.